messaging/docker-compose.yml

```diff
@@ -29,6 +29,12 @@ services:
       - stos-network
     environment:
       - RABBIT_ADDR=rabbitmq
+  worker:
+    build:
+      context: .
+      dockerfile: ./worker/Dockerfile
```

Comment:
Po co to jest robione build contextem jak nie kopiujesz zadnych plikow z parent folderu?